docs/UserGuide.adoc

```diff
@@ -243,14 +248,15 @@ _(If recipe 2 specifies 500 ml of milk, it will be subtracted and updated to 300
 
 This command allows you to add a preparation step to the selected recipe.
 Requires a valid index in the list of recipes and the cooking step.
+The STEP_INDEX given must be at most 1 more than the current number of steps currently in the recipe.
```

Comment:
An example would help bring out the point here

Reply:
Thanks for looking out! Pushed a PR for all the changes for this and the above